src/Http/src/Config/HttpConfig.php

```diff
@@ -55,9 +55,15 @@ public function getMiddleware(): array
 
     /**
      * If chunkSize isn't provided - using default values
+     *
+     * @psalm-return positive-int
      */
     public function getChunkSize(): ?int
     {
-        return !\is_null($this->config['chunkSize']) ? (int) $this->config['chunkSize'] : null;
+        if (\is_null($this->config['chunkSize']) || (int) $this->config['chunkSize'] <= 0) {
```

Comment:
```suggestion
        if (\is_null($this->config['chunkSize']) || (int) $this->config['chunkSize'] < 0) {
```

I propose to support `0` value. We need to update SapiEmitter also:
```php
// SapiEmitter::emitBody
if ($this->bufferSize === 0) {
    return (string) $body;
}
while (!$body->eof()) {
// ...
```
And need to add comment for the `SapiEmitter::$bufferSize` property about the `0` value that means disabled streamed response

Reply:
Done.